src/pages/profile.js

```diff
@@ -6,20 +7,43 @@ import ProfileBadge from '../components/profile-badge/profile-badge';
 import ProfileContent from '../components/profile-content/profile-content';
 import ProfileFilterMenu from '../components/profile-filter-menu/profile-filter-menu';
 import GitHubIcon from '-!svg-react-loader!../images/github.svg';
+import DisqusIcon from '-!svg-react-loader!../images/disqusIcon.svg';
 import { useAuth0 } from '@auth0/auth0-react';
+import { GetUser } from '../services/apiService';
 
 const Profile = ({ data, gitHubUsername }) => {
   const [selectedFilter, setSelectedFilter] = useState(4);
   const [listChange, setListChange] = useState(0);
-  const { user, isAuthenticated, loginWithRedirect } = useAuth0();
+  const {
+    user,
+    isAuthenticated,
+    loginWithRedirect,
+    getIdTokenClaims,
+  } = useAuth0();
+  const [commentsConnected, setCommentsConnected] = useState(false);
   const [bookmarkedRulesCount, setBookmarkedRulesCount] = useState();
   const [superLikedRulesCount, setSuperLikedRulesCount] = useState();
   const [likedRulesCount, setLikedRulesCount] = useState();
   const [dislikedRulesCount, setDislikedRulesCount] = useState();
   const [superDislikedRulesCount, setSuperDislikedRulesCount] = useState();
   const [commentedRulesCount, setCommentedRulesCount] = useState();
 
-  useEffect(() => {}, [listChange, user]);
+  async function CheckUser() {
+    const jwt = await getIdTokenClaims();
+    GetUser(user.sub, jwt.__raw)
+      .then((success) => {
+        setCommentsConnected(success.commentsConnected);
```

Comment:
Does this need to check for the message returned when the user was already connected?

Also does it need to check for response types and process accordingly (e.g. conflict)?

Reply:
I converted all the error logging to AppInsights

And it will should the user an error message if their is a conflict